app/swagger/requests/appeals.rb

```diff
@@ -60,6 +60,81 @@ class Appeals
         end
       end
 
+      swagger_path '/services/appeals/v0/appeals/higher_level_reviews' do
```

Comment:
When I  [view the swagger docs locally](https://github.com/department-of-veterans-affairs/vets-api/tree/master/app/swagger), I'd expect to see a json schema. ![Screen Shot 2019-12-31 at 12 38 14 PM](https://user-images.githubusercontent.com/19188/71629278-9ddb3b00-2bca-11ea-809b-9fa081f0dd29.png)